opm/parser/eclipse/EclipseState/Eclipse3DProperties.cpp

```diff
@@ -449,6 +449,17 @@ namespace Opm {
         }
     }
 
+    std::vector< int > Eclipse3DProperties::getRegions( const std::string& kw ) const {
```

Comment:
Can this be renamed to e.g. `getRegionValues()` - to more clearly differentiate from the full region keyword itself.